api/test/unit/validation/schemaValidator.test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var expect = require("chai").expect;
```

Comment:
That's right. We debated before we decided that we need this test to guard against anyone changing anything in the methods which may impact the policy validation indirectly.